bff/src/sse/sse.ts

```diff
@@ -0,0 +1,53 @@
+import helper from "../utils/helper";
+import * as express from "express";
+import { createClient } from "redis";
+import EventEmitter = require("events");
+
+const event = new EventEmitter();
+
+async function startRedis() {
+  console.log('Starting Redis');
+  const client = createClient({
+    url: "redis://cloudknit-redis-master.zlifecycle-system.svc.cluster.local:6379"
+  });
+
+  client.on("error", (err) => console.log("Redis Client Error", err));
+
+  await client.connect();
+  client.SUBSCRIBE("test-channel", (message: any, channel: any) => {
```

Comment:
Should this channel name be configurable? Should it be keyed off the Organization?